src/main/java/com/google/songgame/servlets/ChatServlet.java

```diff
@@ -80,10 +80,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String message = data.get("message");
     String messageType = "guess";
 
-    if (checkGuessStatus(userId, currentRound)) {
+    if (checkIfUserGuessedCorrect(userId, currentRound)) {
```

Comment:
sorry to be annoying with the naming but now i think we have another issue, the checkGuess name is ambiguous

theres 3 things here

checkIfUserPreviouslyGuessedCorrect

checkCorrectGuess(message)

markUserGuessedCorrectly

something along these lines. let me know if this makes sense